plugins/experimental/ts_lua/ts_lua_http.c

```diff
@@ -29,9 +29,33 @@ typedef enum {
   TS_LUA_CACHE_LOOKUP_SKIPPED
 } TSLuaCacheLookupResult;
 
+typedef enum {
+  TS_LUA_SRVSTATE_STATE_UNDEFINED,
+  TS_LUA_SRVSTATE_ACTIVE_TIMEOUT,
+  TS_LUA_SRVSTATE_BAD_INCOMING_RESPONSE,
+  TS_LUA_SRVSTATE_CONNECTION_ALIVE,
+  TS_LUA_SRVSTATE_CONNECTION_CLOSED,
+  TS_LUA_SRVSTATE_CONNECTION_ERROR,
+  TS_LUA_SRVSTATE_INACTIVE_TIMEOUT,
+  TS_LUA_SRVSTATE_OPEN_RAW_ERROR,
+  TS_LUA_SRVSTATE_PARSE_ERROR,
+  TS_LUA_SRVSTATE_TRANSACTION_COMPLETE,
+  TS_LUA_SRVSTATE_CONGEST_CONTROL_CONGESTED_ON_F,
+  TS_LUA_SRVSTATE_CONGEST_CONTROL_CONGESTED_ON_M
+} TSLuaServerState;
+
 char *ts_lua_cache_lookup_result_string[] = {"TS_LUA_CACHE_LOOKUP_MISS", "TS_LUA_CACHE_LOOKUP_HIT_STALE",
                                              "TS_LUA_CACHE_LOOKUP_HIT_FRESH", "TS_LUA_CACHE_LOOKUP_SKIPPED"};
 
+char *ts_lua_server_state_string[] = {"TS_LUA_SRVSTATE_STATE_UNDEFINED", "TS_LUA_SRVSTATE_ACTIVE_TIMEOUT"
+                                                                         "TS_LUA_SRVSTATE_BAD_INCOMING_RESPONSE",
+                                      "TS_LUA_SRVSTATE_CONNECTION_ALIVE"
+                                      "TS_LUA_SRVSTATE_CONNECTION_CLOSED",
+                                      "TS_LUA_SRVSTATE_CONNECTION_ERROR", "TS_LUA_SRVSTATE_INACTIVE_TIMEOUT",
+                                      "TS_LUA_SRVSTATE_OPEN_RAW_ERROR", "TS_LUA_SRVSTATE_PARSE_ERROR",
+                                      "TS_LUA_SRVSTATE_TRANSACTION_COMPLETE", "TS_LUA_SRVSTATE_CONGEST_CONTROL_CONGESTED_ON_F",
+                                      "TS_LUA_SRVSTATE_CONGEST_CONTROL_CONGESTED_ON_M"};
```

Comment:
Add a ',' before the closing brace to encourage ``clang-format` to format this right.